agent_application/docker/README.md

```diff
@@ -12,17 +12,34 @@ docker build -f docker/Dockerfile -t ssi-agent ..
 
 Inside the folder `/agent_application/docker`:
 
-1. Inside `docker-compose.yml` replace the environment value: `AGENT_APPLICATION_URL` with your actual local ip address or url (such as http://192.168.1.234:3033)
+1. Inside `docker-compose.yml` replace the environment value: `AGENT_APPLICATION_URL` with your actual local IP address or URL (such as http://192.168.1.234:3033)
 2. Optionally, add the following environment variables:
    - `AGENT_ISSUANCE_CREDENTIAL_NAME`: To set the name of the credentials that will be issued.
    - `AGENT_ISSUANCE_CREDENTIAL_LOGO_URL`: To set the URL of the logo that will be used in the credentials.
-3. To start the **SSI Agent**, a **Postgres** database along with **pgadmin** (Postgres Admin Interface) simply run:
+3. By default, UniCore will automatically generate a temporary secure Stronghold file which will be used to sign authorization
+   requests and credentials. Note that using this default option, this Stronghold file will NOT be persisted. If you
```

Comment:
I would actually make this fact a little more outstanding since it's very important to know. You can [use the official "Alert" syntax](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts).